src/lib/components/common/Button.svelte

```diff
@@ -0,0 +1,64 @@
+<script lang="ts">
+  export let href: string = null;
+  export let primary = false;
+  export let ghost = false;
```

Comment:
What do you think about making this an enum instead of overlapping booleans, like:
```
export let mode: 'standard' | 'primary' | 'ghost' = 'standard';
```
In this case, `ghost` would be a dedicated button mode, not a condition. This feels more specific to me, and helps keep the API cleaner if we add more modes over time. (See existing button for mode overspeccing).